modules/Movecraft/src/main/java/net/countercraft/movecraft/commands/SiegeCommand.java

```diff
@@ -189,6 +224,22 @@ private void startSiege(Siege siege, Player player, long cost) {
         siege.setStage(SiegeStage.PREPERATION);
     }
 
+    private void cancelSiege(Siege siege) {
+        @NotNull Player siegeLeader = Movecraft.getInstance().getServer().getPlayer(siege.getPlayerUUID());
+        Bukkit.getServer().broadcastMessage(String.format(I18nSupport.getInternationalisedString("Siege - Siege Failure"),
+                siege.getName(), siegeLeader.getDisplayName()));
+
+        siege.setStage(SiegeStage.INACTIVE);
+
+        List<String> commands = siege.getCommandsOnWin();
```

Comment:
Do you mean to call `getCommandOnLose()` ?

Reply:
Good catch, I'll fix that quick;ly